packages/cli/src/admin-sdk.ts

```diff
@@ -0,0 +1,224 @@
+import { createClient } from './autogen/client/createClient';
+import fse from 'fs-extra';
+import { kebabCase, omit } from 'lodash';
+import chalk from 'chalk';
+import { readAsJson, getFiles, getDirectories, replaceField } from './utils';
+import cliProgress from 'cli-progress';
+import { createHash } from 'crypto';
+import traverse from 'traverse';
+
+const multibar = new cliProgress.MultiBar(
+  {
+    clearOnComplete: false,
+    hideCursor: true,
+    format: '|{bar}| {name} | {value}/{total}',
+  },
+  cliProgress.Presets.shades_grey
+);
+
+// todo change this to prod
+const root = 'https://qa.builder.io';
+
+export const importSpace = async (privateKey: string, directory: string, debug = false) => {
+  const graphqlClient = createClient({
+    fetcher: ({ query, variables }, fetch, qs) =>
+      fetch(`${root}/api/v2/admin?${qs.stringify({ query, variables })}`, {
+        headers: {
+          'Content-Type': 'application/json',
+          Authorization: `Bearer ${privateKey}`,
+        },
+      }).then(r => r.json()),
+  });
```

Comment:
Maybe refactor this into it's own thing? So this could end up being just:
```suggestion
  const graphqlClient = createBuilderGraphQLClient(privateKey);
```